demo/view-timeline/index.html

```diff
@@ -110,7 +123,23 @@
     <div class="progress-bar-progress" style="top: 170px;"></div>
     <div class="progress-bar-progress" style="top: 220px;"></div>
     <div class="progress-bar-progress" style="top: 270px;"></div>
+    <div class="progress-bar-progress" style="top: 320px;"></div>
```

Comment:
I'm a bit worried that this demo is getting too busy. 10 progress bars feels like a lot.  How about rename the current demo to indicate it's for view-timeline delays (timeRange was a placeholder and it appears we will use delay and endDelay instead).  For insets, we could have a nice visual with a box smaller than the viewport by the inset amount and show that the animated range aligns with the bounds of the inset box.